tests/jballerina-unit-test/src/test/resources/test-src/types/constant/simple-literal-constant-negative.bal

```diff
@@ -321,3 +321,35 @@ Ints2 ints2 = 2;
 
 float:NaN floatNan = 1.0;
 float:Infinity floatInf = 1.0;
+
+// Test arithmatic errors
```

Comment:
```suggestion
// Test arithmetic errors
```

Reply:
We can add this comment in line no#331 and get rid of `//error` comment in the lines which have tested the arithmetic operations